README.md

```diff
@@ -110,6 +110,14 @@ higher-level integrations with UI frameworks.
 * [The Reactive Workflow Pattern Update – Droidcon SF 2017](https://www.youtube.com/watch?v=mvBVkU2mCF4)
 * [The Rx Workflow Pattern – Droidcon NYC 2017](https://www.youtube.com/watch?v=KjoMnsc2lPo)
   ([slides](https://speakerdeck.com/rjrjr/reactive-workflows))
+  
+### Support & Contact
+
+Workflow maintainers hang out in the [#squarelibraries](https://kotlinlang.slack.com/messages/C5HT9AL7Q) channel on the [Kotlin Slack](https://surveys.jetbrains.com/s3/kotlin-slack-sign-up?_ga=2.93235285.916482233.1570572671-654176432.1527183673) and the [#square-libraries-wtf](https://androidstudygroup.slack.com/messages/C03NYGB45) channel on the Android Study Group Slack. We also have a separate Slack space dedicated to Workflow for design discussions. If you'd like to join, just reach out to any of us on Twitter or wherever and ask!
```

Comment:
I'm okay with the two slack links, but I'd drop the rest, including the hint about the private slack.

@bencochran I don't imagine you know of any comparable Swift / iOS Slacks we should be haunting?

Reply:
Done.